setup.py

```diff
@@ -24,14 +24,14 @@ def read_file(filename):
 
 
 about = {}
-exec(read_file(os.path.join("src", "aiohttp_cors", "__about__.py")), about)
+exec(read_file(os.path.join("aiohttp_cors", "__about__.py")), about)
 
 needs_pytest = {'pytest', 'test'}.intersection(sys.argv)
 pytest_runner = ['pytest_runner'] if needs_pytest else []
 
-# aiohttp requires Python >= 3.4, so as aiohttp_cors.
-if sys.version_info[:2] < (3, 4):
-    print("Error: aiohttp_cors requires Python interpreter version >= 3.4, "
+# aiohttp requires Python >= 3.4.1, so as aiohttp_cors.
```

Comment:
@asvetlov Actually I want to change back this check to require Python 3.4.

I lifted required Python version because I used [subTest](https://docs.python.org/3.5/library/unittest.html#distinguishing-test-iterations-using-subtests) feature of unittest that introduced in Python 3.4.
Actual required Python will be, of course, Python >=3.4.1, because this what aiohttp requires, and aiohttp_cors depends on aiohttp.

My comment here is incorrect: "aiohttp requires Python >= 3.4, so as aiohttp_cors.", I explained reason why it's allowed to update Python version, but not why I want to update it.